### ISSUE TITLE
New rule to prevent to use `expect(query*(...)).toBeDefined();`

### ISSUE THREAD

**Issue by retyui** [jest-community/eslint-plugin-jest#1422]
False Positives check, that need to be forbidden:

```tsx
query*(...) // null | Elemnt
expect(null).toBeDefined(); // true
```

**Comment by retyui** [comment]
sorry miss prj, should be RNTL eslint issue
